tests/test_json.py

```diff
@@ -97,6 +97,29 @@ class Zone(proto.Enum):
     assert json2 == '{"zone":"EPIPELAGIC"}'
 
 
+def test_json_default_values():
+    class Squid(proto.Message):
+        mass_kg = proto.Field(proto.INT32, number=1)
+        name = proto.Field(proto.STRING, number=2)
+
+    s = Squid(name="Steve")
+    json1 = (
+        Squid.to_json(s, including_default_value_fields=False)
+        .replace(" ", "")
+        .replace("\n", "")
+    )
+    assert json1 == '{"name":"Steve"}'
+
+    json2 = Squid.to_json(s).replace(" ", "").replace("\n", "")
+    assert (
+        json2 == '{"name":"Steve","massKg":0}' or json2 == '{"massKg":0,"name":"Steve"}'
```

Comment:
Is the ordering not stable?

Reply:
No but it shouldn't matter for json objects right?

Reply:
I don't think it should matter, I just was curious if you were running into problems during testing.